src/content/en/fundamentals/performance/rail.md

```diff
@@ -106,6 +106,28 @@ of their time waiting for sites to respond to their input, not waiting for the s
   careful not to block the user. If possible, do work in the background.
 * For actions that take longer than 50ms to complete, always provide feedback.
 
+**50ms or 100ms?:**
```

Comment:
One line 94, I think it might make sense to clarify what we mean.

" ## Response: respond in under 50ms {: #response }"

"Respond in under 50ms" makes it sound like the total time taken should be < 50ms, but it just needs to be under 100ms.

I think that section in general could be a bit clearer. Perhaps instead of "respond in under 50ms" it should be "process events in under 50ms"?

Reply:
👍 on Tim's suggestion for tightening up the wording in the respond in <50ms line. Perhaps if we switch to "process events", it's worth including a very brief example so it's clear to folks what we mean too?

Reply:
Changed the heading and also modified the first guideline:

> Process user input events within 50ms to ensure a visible response within 100ms, otherwise the connection between action and reaction is broken. This applies to most inputs, such as clicking buttons, toggling form controls, or starting animations. This does not apply to touch drags or scrolls.